openwisp_monitoring/device/static/check/css/check_inline.css

```diff
@@ -0,0 +1,3 @@
+.inline-group .tabular td.original p {
+    font-size: 14px;
+}
```

Comment:
I would remove this because this will be in our way when we'll do a general redesign of the theme and use different font-sizes (we could use em instead of px for example).

Reply:
It's displayed properly without this after using `GenericStackedInline`, so removed it :smiley: